src/api/app/views/webui/request/beta_show_tabs/_rpm_lint_result.html.haml

```diff
@@ -1,6 +1,11 @@
-%p
-  The RPM Lint results aren't here yet. For now, you can see them on the request's
-  source package page. We're working on having them here in the future.
+#rpmlint-log{ data: { project: project.name, package: package.name } }
+  - unless repository_list.empty?
+    .form-inline
+      = select_tag('rpmlint_repo_select', options_for_select(repository_list.sort),
+                   class: 'custom-select', onchange: 'updateRpmLintArchitectures()')
+      - repo_arch_hash.each do |repository, architectures|
+        = select_tag("rpmlint_arch_select_#{repository}", options_for_select(architectures.reverse),
+                     class: 'rpmlint_arch_select custom-select', onchange: 'updateRpmlintDisplay()')
```

Comment:
If I understand correctly, you moved JS functions related to RPM Lint to an independent file and renamed the functions... so this should be changed:

```suggestion
                     class: 'rpmlint_arch_select custom-select', onchange: 'updateRpmLintLog()')
```

Reply:
Very true. Suggestion applied.Thanks!